src/Ionide.ProjInfo.ProjectSystem/WorkspacePeek.fs

```diff
@@ -45,7 +45,23 @@ module WorkspacePeek =
             let scanDir (dirInfo: DirectoryInfo) =
                 let hasExt (ext: string) (s: FileInfo) = s.FullName.EndsWith(ext)
 
-                dirInfo.EnumerateFiles("*.*", SearchOption.TopDirectoryOnly)
+                let topLevelFiles =
+                    try
+                        dirInfo.EnumerateFiles("*.*", SearchOption.TopDirectoryOnly)
```

Comment:
Does the exception happen on `EnumerateFiles` or when actually enumerated? (like in Seq.choose) 

This makes me think if we hit an exception on any file/folder at the root, we wouldn't explore the rest.

Reply:
I tested it with both a file/directory in the root which required elevated permissions and it seemed to work just fine.